src/test/miner_tests.cpp

```diff
@@ -26,6 +27,25 @@
 
 BOOST_FIXTURE_TEST_SUITE(miner_tests, TestingSetup)
 
+// Validation errors used in tests (prevent typos and facilitate refactoring):
+#define REJECT_INVALID_BAD_BLK_SIGOPS "bad-blk-sigops"
```

Comment:
IMO inline these strings.

Reply:
I used `#define` to make it easier to find all occurrences in the codebase if these rejection reasons ever need to be changed. It also helps against typos, although typos would be immediately obvious when you run the test.

Happy to change it to inline if nobody objects.

Reply:
The test will fail if there is a typo. Having inline is good because a rename forces you to update the relevant tests and double check everything related.

Reply:
Using inline now.